pallets/utxo/src/tests.rs

```diff
@@ -400,3 +400,36 @@ fn test_tokens() {
         assert_ok!(Utxo::spend(Origin::signed(0), tx.clone()));
     });
 }
+    
+#[test]
+fn attack_double_spend_by_tweaking_input() {
```

Comment:
This is @iljakuklic 's code right? The stuff that's just been merged? It looks identical but there might be a slight difference I'm missing

Reply:
There is a commit that merges in `staging` but I am not sure why this shows up in the diff

Reply:
@muursh I've just merged it from staging. Should be correct but let @iljakuklic recheck that.

Reply:
> There is a commit that merges in staging but I am not sure why this shows up in the diff

Okay, if it is correct, let's resolve this